docs/Converters/UKBtoBIDS.md

```diff
@@ -0,0 +1,86 @@
+<!-- markdownlint-disable MD046 -->
+# `ukb-to-bids` – Conversion of the UK Biobank (UKB) to BIDS
+
+!!! quote "Description reproduced from the [UK Biobank webpage](https://www.ukbiobank.ac.uk/)"
+    UK Biobank is a large-scale biomedical database and research resource, containing in-depth genetic and health information from half a million UK participants. The database is regularly augmented with additional data and is globally accessible to approved researchers undertaking vital research into the most common and life-threatening diseases. It is a major contributor to the advancement of modern medicine and treatment and has enabled several scientific discoveries that improve human health.
+
+## Dependencies
+
+If you installed the core of Clinica, this converter needs dcm2niix.
+
+## Downloading UK Biobank
+
+The UK Biobank to BIDS converter requires the user to have applied to the data and to have downloaded the UK Biobank imaging and clinical data.
+
+Once you have access to it, the imaging data can be downloaded and extracted using this tutorial: https://biobank.ctsu.ox.ac.uk/~bbdatan/Accessing_UKB_data_v2.3.pdf, and the clinical data can be extracted using the information from this repository https://github.com/kenhanscombe/ukbtools.
+
+Be careful, the converter only takes in the nifti provided by UK Biobank, not the dicoms.
+Also, the clinical_data.tsv needs to contain these informations about the subject: sex, year of birth, age at recruitment, age at sessions.
+
+## Supported modalities
+
+Please note that this converter processes the following modalities : 
+- T1W (nifti)
+- T2 Flair (nifti)
+- DWI (nifti)
+- SWI (nifti)
+- tfMRI (dicom)
+- rsfMRI (dicom)
+
+!!! Chosen files
+    Whenever possible, we use the rawest files so as to leave it to the user to choose the processing needed. When available we get the associated json.
+| Modality    | Chosen image(s) | Justification |
+| :----------:|:---------------:|:-------:|
+| T1W                     | T1.nii.gz       | Defaced and cropped so there is no neck. The rawest image would be the simply defaced one, but since we have no interest in the neck we don't use it. We do not want any other corrections. |
+| T2 Flair    | T2_FLAIR.nii.gz | Defaced and cropped so there is no neck. The reasons for choosing this image are the same as for T1. |
+| DWI         | AP/PAnii.gz     | It is the rawest images we can get. bvals and bvec are also available. |
+| rsfMRI      | rsfMRI.dcm | Since the nii.gz version doesn't always include a json and in consideration for it's usage, we convert the dicom. |
```

Comment:
"in consideration for it's usage" sounds a bit awkward --> "The dicom sources are used to extract BIDS required metadata not found otherwise"